setup.py

```diff
@@ -0,0 +1,30 @@
+from setuptools import setup
+
+setup(
+    name='edx-repo-tools',
+    version='0.1',
+    packages=[
+        'edx_repo_tools',
+        'edx_repo_tools.oep2',
+        'edx_repo_tools.oep2.checks',
+    ],
+    install_requires=[
```

Comment:
Good question. This is mainly for when you're installing `oep2` into another repo, so I wanted more relaxed versions. I'm not aware of any particular breakages for specific version ranges, but if we hit incompatibilities, we'll want to bracket those in this file.